openforcefield/tests/test_smirnoff.py

```diff
@@ -112,42 +112,42 @@
 ffxml_contents_gbsa = u"""\
 <?xml version="1.0"?>
 
-<SMIRFF use_fractional_bondorder="True">
+<SMIRNOFF use_fractional_bondorder="True">
 
 %(ffxml_standard)s
 %(ffxml_constraints)s
 %(ffxml_gbsa)s
 
-</SMIRFF>
+</SMIRNOFF>
 """ % globals()
 
 ffxml_contents_noconstraints = u"""\
 <?xml version="1.0"?>
 
-<SMIRFF use_fractional_bondorder="True">
+<SMIRNOFF use_fractional_bondorder="True">
 
 %(ffxml_standard)s
 
-</SMIRFF>
+</SMIRNOFF>
 """ % globals()
 
 ffxml_contents = u"""\
 <?xml version="1.0"?>
 
-<SMIRFF use_fractional_bondorder="True">
+<SMIRNOFF use_fractional_bondorder="True">
 
 %(ffxml_standard)s
 %(ffxml_constraints)s
 
-</SMIRFF>
+</SMIRNOFF>
 """ % globals()
 
 # Set up another, super minimal FF to test that alternate aromaticity
 # models implemented properly
 ffxml_MDL_contents = u"""\
 <?xml version="1.0"?>
 
-<SMIRFF version="0.1" aromaticity_model="OEAroModel_MDL">
+<SMIRNOFF version="0.1" aromaticity_model="OEAroModel_MDL">
 
 !-- SMIRKS (SMIRKS Force Field) minimal file, not intended for use -->
```

Comment:
All "SMIRKS Force Fields" should have the native open added to it. I don't think I caught all of these in smarty or smirnoff99Frosst